src/status_im/contexts/chat/messenger/placeholder/view.cljs

```diff
@@ -16,10 +16,11 @@
 (defn view
   [on-layout-done?]
   (let [theme       (quo.theme/use-theme)
+        screen-id   (quo.theme/use-screen-id)
         chat-exist? (rf/sub [:chats/current-chat-exist?])]
     [rn/view {:style (style/container theme @on-layout-done?)}
      (when-not chat-exist?
        [quo/page-nav
         {:icon-name :i/arrow-left
-         :on-press  #(rf/dispatch [:navigate-back])}])
+         :on-press  #(rf/dispatch [:navigate-back screen-id])}])
```

Comment:
could pls elaborate on this change

Reply:
Related to https://github.com/status-im/status-mobile/pull/22346/files#r2009923719.

We are using screen-id here to close chat in ios ([view-id might be wrong](https://github.com/status-im/status-mobile/issues/21659))
(original report - https://github.com/status-im/status-mobile/pull/21643#issuecomment-2488962044)

>  Seems odd we do two different types of detections, one for iOS and another for Android. ([ref](https://github.com/status-im/status-mobile/pull/22336/files#r2003671117))

This is approch to have chat/close solution at one place, another approach - https://github.com/status-im/status-mobile/pull/22336/files